internal/demo/util/client.go

```diff
@@ -38,10 +38,6 @@ func newClient(buildID string, metricsPort int) (c client.Client, stopFunc func(
 		panic(err)
 	}
 
-	if _, err := c.CheckHealth(context.Background(), &client.CheckHealthRequest{}); err != nil {
```

Comment:
I think this should work. What is the error? If it doesn't work, we should understand why and replace it with a better health check.